javascript/react/src/verbs/Convert.base.tsx

```diff
@@ -46,7 +43,10 @@ export const ConvertBase: React.FC<
 				label: 'Delimiter',
 				if:
 					step.args.type === ParseType.Array ||
-					isInputColumnArray(fields, step.args.columns),
+					isInputColumnArray(
```

Comment:
I'm not sure this is quite how to do this anymore. The utility should just accept a single column now, and check if the data type === DataType.Array